flowey/flowey_lib_hvlite/src/artifact_rustdoc.rs

```diff
@@ -38,9 +39,8 @@ pub mod publish {
                 |rt| {
                     let rustdocs_dir = rt.read(rustdocs_dir);
                     let sh = xshell::Shell::new()?;
-                    // use in-box tar, which supports zip. works for all
-                    // windows builds past Windows 10 build 17063
-                    xshell::cmd!(sh, "tar -a -c -f rustdoc.zip {rustdocs_dir}").run()?;
+                    let bsdtar = bsdtar_name(rt);
```

Comment:
Rather than having a `bsdtar_name` freefloating function, this should probably be backed by a `ctx.reqv(install_bsdtar::Request)`, which return a `PathBuf` to the bsdtar executable.

that way, we can evolve the underlying `install_bsdtar` node to install the dependency on platforms where it might not be present in-box in the future.

In the future, I'd also want to hook that into flowey's as-yet-unrealized-and-not-yet-designed "platform requirements" API, where I'd like nodes to be able to communicate requests to the pipeline resolver in order to have certain dependencies pre-installed on the job runner (similar to ADO's "demands" system they have for pools)

Reply:
Sure, seems worth opening an issue. Or I can revert this particular change.

Reply:
We can track this in a follow up

Reply:
#147